controllers/topics-controllers.js

```diff
@@ -1,4 +1,14 @@
-const { selectTopics } = require("../models/topics-models");
+const { readDoc, selectTopics } = require("../models/topics-models");
+
+exports.getDoc = (request, response, next) => {
+  readDoc()
```

Comment:
Rather than having to invoke a model here, we can simply require in the endpoints.json and send this as the response 👍